File: DefaultCodeInputOutput/2.js
function lengthOfLongestSubstring(s) {
  //Write your code here
}

function main() {
  if (process.argv.length !== 3) {
      console.log("Please provide a single JSON string argument.");
      return;
  }

  let inputJson = process.argv[2];
  let inputData;
  try {
      inputData = JSON.parse(inputJson);
  } catch (e) {
      console.log("Invalid JSON input.");
      return;
  }

  let s = inputData.s || "";
  let result = lengthOfLongestSubstring(s);
  let output = { result: result };
  console.log(JSON.stringify(output));
}

main();

File: DefaultCodeInputOutput/2.py
import sys
import json

def length_of_longest_substring(s: str) -> int:
   #write your code here

    return max_length

def main():
    if len(sys.argv) != 2:
        print("Please provide a single JSON string argument.")
        return

    input_json = sys.argv[1]
    try:
        input_data = json.loads(input_json)
        s = input_data.get("s", "")
    except json.JSONDecodeError:
        print("Invalid JSON input.")
        return

    result = length_of_longest_substring(s)
    output = {"result": result}
    print(json.dumps(output))

if __name__ == "__main__":
    main()
